src/rise-twitter.js

```diff
@@ -110,6 +109,7 @@ export default class RiseTwitter extends HTMLElement {
       this.tweet = new Tweet(this.shadowRoot, this.logger, this.settings, this.eventHandler, this.state);
       this.messaging = new Messaging(this.tweet, this.id, this.localMessaging, this.config, this.settings, this.logger);
       this.screenName = event.detail.screenName;
+      this.id = event.detail.componentId;
```

Comment:
This should probably moved with screenName to the beginning of the block since this.id is used in other "classes" that are init before this.